src/ext/java/org/opentripplanner/ext/transmodelapi/model/siri/sx/PtSituationElementType.java

```diff
@@ -175,16 +175,19 @@ public static GraphQLObjectType create(
           .name("summary")
           .type(new GraphQLNonNull(new GraphQLList(new GraphQLNonNull(multilingualStringType))))
           .description("Summary of situation in all different translations available")
-          .dataFetcher(environment -> {
-            I18NString headerText = environment.<TransitAlert>getSource().headerText();
-            if (headerText instanceof TranslatedString translatedString) {
-              return translatedString.getTranslations();
-            } else if (headerText != null) {
-              return List.of(new AbstractMap.SimpleEntry<>(null, headerText.toString()));
-            } else {
-              return emptyList();
-            }
-          })
+          .dataFetcher(environment ->
+            environment
+              .<TransitAlert>getSource()
+              .headerText()
+              .map(headerText -> {
+                if (headerText instanceof TranslatedString translatedString) {
+                  return translatedString.getTranslations();
+                } else {
+                  return List.of(new AbstractMap.SimpleEntry<>(null, headerText.toString()));
+                }
+              })
+              .orElse(emptyList())
```

Comment:
We may want to standardize on one construct in the code? But the technical differences is somewhat of a rabbit hole...
```suggestion
              .orElse(List.of())
```

Reply:
I'm fine with that. Maybe it could be associated to the issue of serialization?